### PR TITLE
Add nodePort to helm service.yaml

### PR DESCRIPTION
**Pull request by gauthier-labs**
Adding nodePort to the service.yaml enables a static external port to access the LakeFS GUI.

This is good for smaller deployments and testing.

### CODE REVIEW

File: charts/lakefs/templates/service.yaml [hunk 1]
```diff
@@ -9,6 +9,9 @@ spec:
   ports:
     - port: {{ .Values.service.port }}
       targetPort: http
+{{- if .Values.service.nodePort }}
```
Comment (Isan-Rivkin) [inline charts/lakefs/templates/service.yaml:12]: please keep the `if` statement at the same ident level as the field itself

Comment (Isan-Rivkin) [inline charts/lakefs/templates/service.yaml:12]: please assert that `.Values.service.type == "NodePort"` to avoid issues for users configuring this.

**Review by Isan-Rivkin — CHANGES_REQUESTED** [review]
Hey, thanks for your PR. 
Added few comments + notice the CI checks :) 
Thanks.

File: charts/lakefs/templates/service.yaml (hunk 1 above)
Comment (gauthier-labs) [reply charts/lakefs/templates/service.yaml]: Updated indentation

Comment (gauthier-labs) [reply charts/lakefs/templates/service.yaml]: Added check for .Values.service.type

**Comment by gauthier-labs** [comment]
@Isan-Rivkin Your requested changes have been made. Awaiting your conversation resolutions.

**Comment by Isan-Rivkin** [comment]
@gauthier-labs thanks for the update! Looks good! 
Please notice since then the `Chart.yaml` version has be already upgraded, make sure to align it to next version (since a merge to main will release it) and I'll approve and merge :)

**Review by Isan-Rivkin — APPROVED** [review]
LGTM @gauthier-labs ! 
Thank you for the patience and contribution!

**Pull request merged**